### DISCUSSION TITLE
How to change binding adapter IP

### DISCUSSION THREAD

**Discussion by wop86** [Drakkar-Software/OctoBot#1910] [Q&A] [answered]
When executed on a Ubuntu VPS, I can change the web port but not the IP/Adapter in witch the web server is listening. Is there a config statement to put in the config.json?

**Comment by Herklos** [comment, marked as answer]
Hi @wop86, yes you can set the environment var `WEB_PORT` and `WEB_ADDRESS` to set the host IP and the port you want to expose your web interface.

**Reply by wop86** [reply]
Thanks a lot Herklos!  That's a fast reply ;-)
